stdlib/2and3/lib2to3/pygram.pyi

```diff
@@ -0,0 +1,107 @@
+# Stubs for lib2to3.pygram (Python 3.6)
+
+from typing import Any
+from lib2to3.pgen2.grammar import Grammar
+
+class Symbols:
+    and_expr: int
```

Comment:
These mostly aren't guaranteed to exist, since the class just setattrs whatever attributes exist on the grammar you pass in. Are the concrete names useful to you? We could also add a `def __getattr__(self, attr: name) -> int` or something to indicate that it has arbitrary attributes.